### ISSUE TITLE
Release action is failing to upload artifacts

### ISSUE THREAD

**Issue by rasaha91** [facebook/flipper#4182]
## 🐛 Bug Report
See https://github.com/facebook/flipper/actions/runs/3150989565/jobs/5124580334 where we are having a failure to upload release artifacts AND failing to open an issue for this failure, which is why there is no issue tracking it.

## To Reproduce

N/A

## Environment

N/A

**Comment by rasaha91** [comment]
I debugged this issue and I believe the problem is the fact that https://github.com/passy/github-upload-release-artifacts-action uses a docker image based on Alpine linux, which uses a minimal version of libc. The error messages in the failed android publishing and release workflows is that ghr cannot be found, despite downloading it to /usr/local/bin. I verified that /usr/local/bin/ghr does exist. I ran ldd to see what the issue was (see https://github.com/rasaha91/flipper/actions/runs/3238534082/jobs/5306864002#step:2:112). This is effectively the same issue in this SO thread: https://stackoverflow.com/questions/67672392/docker-image-with-alpine-linux-an-executable-file-is-definitely-there-but-canno

I don't know why this would be happening all of a sudden as the dockerfile for the action hasn't been updated since November 2021. 

@passy and @lblasa in case you have ideas.

**Comment by rasaha91** [comment]
Upon further inspection, we download the latest release of ghr. This release: https://github.com/tcnksm/ghr/releases/tag/v0.16.0 was released 17 days ago, and the release action started failing around 14 days ago, so the timeline seems right. My guess is something breaking on Alpine linux was introduced in that release. Perhaps a simple fix would be to hardcode the release tag to download.

**Comment by rasaha91** [comment]
@aigoncharov hoping to get some traction on this issue.

**Comment by rasaha91** [comment]
I verified that using v15.0 of ghr works. That version is statically linked, which is why we don't have any problems.

**Comment by rasaha91** [comment]
PR for the fix is above. @passy  can you please merge the PR and create a new release?

**Comment by rasaha91** [comment]
Upstream PR is here: https://github.com/Roang-zero1/github-upload-release-artifacts-action/pull/5

**Comment by aigoncharov** [comment]
@rasaha91 thanks for reporting the issue and coming up with a fix! @passy is on a leave though. I've [forked your fork](https://github.com/aigoncharov/github-upload-release-artifacts-action) (xD) to get unblocked.
Many thanks for making Flipper better!

**Comment by aigoncharov** [comment]
Should be resolved by af4ecc0a941a3bd7e851b2549bb494bc9c1d424c

**Comment by rasaha91** [comment]
Thanks @aigoncharov. Can we manually redo the last 3 releases so the apps are included?

**Comment by aigoncharov** [comment]
> Thanks @aigoncharov. Can we manually redo the last 3 releases so the apps are included?

Do you think it is worth it? I wanted to just push out a new release on Monday

**Comment by rasaha91** [comment]
Someone had requested it in another issue that was a dupe of this one. It's up to you.
